### DISCUSSION TITLE
Selecting the insert block option replaces the current block

### DISCUSSION THREAD

**Discussion by JaniceManwiller** [GitbookIO/community#531] [0. General bugs and questions]
When I click the + icon to add a new block below the current block, and select the block type, instead of adding the new block, it replaces the current block with an empty version of the block that I selected. 

So before I start, the content is:

![AddBlockBefore](https://github.com/GitbookIO/community/assets/107077736/84b035bb-29c8-4141-94df-5c17b60c6e34)

I then click the + sign to insert a paragraph block below the first paragraph:

![AddBlockSelectInsert](https://github.com/GitbookIO/community/assets/107077736/9db49860-e8ca-4ffd-9e32-729300031e97)

When I select Paragraph, the first paragraph is removed and replaced with an empty paragraph:

![AddBlockAfter_EmptyBlock](https://github.com/GitbookIO/community/assets/107077736/aaa4d5a6-1590-4045-bc8c-200d609796e6)

The same thing happens if I use Option-+ to add a block above the current block.

**Comment by rodrcastro** [comment]
Hey @JaniceManwiller! Our team is aware of this and they have already fixed this. It's going to be released soon, keep an eye out for the changes ;)

**Comment by aikoii** [comment]
👋 @JaniceManwiller, I'm going to close this out since we've resolved the issue. Thanks again for flagging!
